install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -52,6 +52,25 @@ backend:
 {{- end -}}
```

Comment:
I assume the commit message should be `The CronJob object reached GA in Kubernetes 1.21` (instead of 1.2)

Reply:
Correct. Will fix.